.github/workflows/main.yml

```diff
@@ -11,6 +11,22 @@ on:
   pull_request:
 
 jobs:
+  close-issues:
```

Comment:
Now that we added a new workflow in `close-inactive-issues.yml` I guess we should remove the configuration in this file (`main.yml`).

Reply:
done, thanks for a nice catch!